### ISSUE TITLE
TypeError on startup. error: The "path" argument must be of type string. Received type undefined

### ISSUE THREAD

**Issue by duckness** [Nexus-Mods/Vortex#6053]
**Describe the bug**
Starting Vortex v1.2.2, it will throw an exception consistently.

**To Reproduce**
1. Startup Vortex
2. Exception

**Screenshots**
![Vortex_2020-03-28_12-15-42](https://user-images.githubusercontent.com/6380270/77814675-52172f80-70ee-11ea-8487-e8fcb61cedcc.png)

**Platform (please complete the following information):**
 - OS: Windows 10 1903
 - Vortex Version v1.2.2

**Additional context**

```
Sat, 28 Mar 2020 04:15:28 GMT - error: unrecoverable error message=The "path" argument must be of type string. Received type undefined, title=undefined, subtitle=undefined, stack=TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received type undefined
    at validateString (internal/validators.js:112:11)
    at Object.join (path.js:375:7)
    at C:\Program Files\Black Tree Gaming Ltd\Vortex\resources\app.asar\renderer.js:2133:63
    at tryCatcher (C:\Program Files\Black Tree Gaming Ltd\Vortex\resources\app.asar\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (C:\Program Files\Black Tree Gaming Ltd\Vortex\resources\app.asar\node_modules\bluebird\js\release\promise.js:547:31)
    at Promise._settlePromise (C:\Program Files\Black Tree Gaming Ltd\Vortex\resources\app.asar\node_modules\bluebird\js\release\promise.js:604:18)
    at Promise._settlePromiseCtx (C:\Program Files\Black Tree Gaming Ltd\Vortex\resources\app.asar\node_modules\bluebird\js\release\promise.js:641:10)
    at _drainQueueStep (C:\Program Files\Black Tree Gaming Ltd\Vortex\resources\app.asar\node_modules\bluebird\js\release\async.js:97:12)
    at _drainQueue (C:\Program Files\Black Tree Gaming Ltd\Vortex\resources\app.asar\node_modules\bluebird\js\release\async.js:86:9)
    at Async._drainQueues (C:\Program Files\Black Tree Gaming Ltd\Vortex\resources\app.asar\node_modules\bluebird\js\release\async.js:102:5)
    at Async.drainQueues (C:\Program Files\Black Tree Gaming Ltd\Vortex\resources\app.asar\node_modules\bluebird\js\release\async.js:15:14), details=code: ERR_INVALID_ARG_TYPE, process=renderer

```

[vortex.log](https://github.com/Nexus-Mods/Vortex/files/4396615/vortex.log)

**Comment by wjling** [comment]
Same error

**Comment by Skeuomorph01** [comment]
Same error after update to 1.2.2. Revert to 1.2.1 and everything works as expected

**Comment by IDCs** [comment]
Thanks guys, this has already been fixed for 1.2.3 - please use 1.2.1 until the new version is out.

Duplicate of #6048
